src/telegrafs/actions/thunks.ts

```diff
@@ -108,6 +108,61 @@ export const createTelegraf = (telegraf: Telegraf) => async (
   }
 }
 
+export const cloneTelegraf = (telegraf: Telegraf) => async (
+  dispatch: Dispatch<Action>
+) => {
+  try {
+    // Step 1: create a new telegraf
+    const response = await postTelegraf({data: telegraf})
+
+    if (response.status !== 201) {
+      throw new Error(response.data.message)
+    }
+
+    const clonedTelegraf = response.data
```

Comment:
I think we should have multiple `try/catch` blocks here, since there are multiple operations. As it is now, if the telegraf cloning passes, but a label cloning fails, we'll dispatch an action that said the telegraf clone failed. With two `try/catch` blocks, we can have separate error messages for each situation, e.g. "we failed to clone your label 'foo', but your telegraf configuration was cloned successfully"